core/crypto/kem/adapter/kem_test.go

```diff
@@ -0,0 +1,30 @@
+package adapter
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/katzenpost/katzenpost/core/crypto/nike/ecdh"
+	"github.com/katzenpost/katzenpost/core/crypto/rand"
+)
+
+func TestNikeToKemAdapter(t *testing.T) {
+	ecdhNike := ecdh.NewEcdhNike(rand.Reader)
+	s := FromNIKE(ecdhNike)
+
+	t.Logf("hello my name is %s", s.Name())
+
+	pubkey1, privkey1, err := s.GenerateKeyPair()
+	require.NoError(t, err)
+
+	ct, ss, err := s.Encapsulate(pubkey1)
+	require.NoError(t, err)
+
+	ss2, err := s.Decapsulate(privkey1, ct)
```

Comment:
using `ss` and `ss2` here is confusing when the functions internally have `ss` and `ss2`. can we call them `ss_a` and `ss_b` here or something like that?

Reply:
I prefer snake case too... but golang is a camel case language. So we'll have to do ssA and ssB